app/filters.php

```diff
@@ -72,15 +72,19 @@
         $comments_template
     );
 
+    $data = collect(get_body_class())->reduce(function ($data, $class) use ($template) {
+        return apply_filters("sage/template/{$class}/data", $data, $template);
+    }, []);
+
     $theme_template = locate_template(["views/{$comments_template}", $comments_template]);
 
     if ($theme_template) {
-        echo template($theme_template);
+        echo template($theme_template, $data);
         return get_stylesheet_directory().'/index.php';
     }
 
     return $comments_template;
-}, 100);
+}, PHP_INT_MAX);
```

Comment:
why is this needed?

Reply:
After render there should not been anything else. If there is 100 some other plugin could modify the path with priority e.g. 200 and that would result in rendering comments twice. When we have render with PHP_INT_MAX, there is a small possibility that other plugin will modify the path after render.

Reply:
After render there should not been anything else. If there is 100 some other plugin could modify the path with priority e.g. 200 and that would result in rendering comments twice. When we have render with PHP_INT_MAX, there is a small possibility that other plugin will modify the path after render.

Reply:
@retlehs Should I keep `PHP_INT_MAX` in here?

Reply:
we don't want to use `PHP_INT_MAX` in the theme /cc @QWp6t

Reply:
What is the reason you don't want to use `PHP_INT_MAX` (for my own information) ? . Thanks.

Reply:
I'm against it because the use of it is too hamfisted and unnecessary. I don't see a reason for us to be that aggressive with the use of that filter or any others.

Reply:
Ok, it's reverted. But generally, I think that after render there should be no chance for a hook with higher priority to render comments again. It's a very same situation like here (https://github.com/roots/sage/blob/master/app/filters.php#L63) where is also render in the filter and `PHP_INT_MAX` priority. I would suggest a consistent approach for both situations, but it's not a scope of this PR and should be in different PR.